File: src/Java/BehavioralPatterns/Mediator/CQRS/src/CreateProductCommand.java
import java.math.BigDecimal;

public class CreateProductCommand implements IRequest {
    private final String name;

    private final BigDecimal price;

    public CreateProductCommand(String name, BigDecimal price) {
        this.name = name;
        this.price = price;
    }

    public String getName() {
        return name;
    }

    public BigDecimal getPrice() {
        return price;
    }
}

File: src/PHP/BehavioralPatterns/Mediator/CQRS/CreateProductCommand.php
<?php

namespace BehavioralPatterns\Mediator\CQRS;

class CreateProductCommand implements IRequest {
    public string $name;
    public float $price;

    public function __construct($name, $price) {
        $this->name = $name;
        $this->price = $price;
    }
}